src/isar/storage/uploader.py

```diff
@@ -0,0 +1,36 @@
+import logging
+from queue import Empty, Queue
+from typing import List
+
+from isar.models.mission_metadata.mission_metadata import MissionMetadata
+from isar.storage.storage_interface import StorageException, StorageInterface
+from robot_interface.models.inspection.inspection import Inspection
+
+
+class Uploader:
+    def __init__(
+        self, upload_queue: Queue, storage_handlers: List[StorageInterface]
+    ) -> None:
+        self.upload_queue: Queue = upload_queue
+        self.storage_handlers: List[StorageInterface] = storage_handlers
+
+        self.logger = logging.getLogger("uploader")
+
+    def run(self) -> None:
+        while True:
+            inspection: Inspection
+            mission_metadata: MissionMetadata
+            try:
+                inspection, mission_metadata = self.upload_queue.get(timeout=5)
+            except Empty:
+                continue
+
+            for storage_handler in self.storage_handlers:
```

Comment:
Do we need to support multiple storage_handlers?

Reply:
Yes. Primary reason being that it might be necessary for us to upload everything to our own storage as a master of records for the data the robot produces. Additionally, analysis in early phases will be a lot easier if we can trigger from our own blob storage.

For testing it is also useful to have the local storage option to be able to inspect quickly what was created.

Reply:
Agreed, the option is good to have